denops_std/option/types.ts

```diff
@@ -0,0 +1,112 @@
+import type { Denops } from "https://deno.land/x/denops_core@v4.0.0/mod.ts";
+
+/**
+ * An option that can be retrieved and modified.
+ * @template T The type of the option value.
+ */
+export interface Option<T> {
```

Comment:
Use `type` here while Option is not required to be extendable.

Reply:
Change to `type`